package.json

```diff
@@ -13,7 +13,15 @@
   "stripes": {
     "type": "settings",
     "displayName": "ui-myprofile.meta.title",
-    "route": "/myprofile"
+    "route": "/myprofile",
+    "links": {
+      "userDropdown": [
+        {
+          "caption" : "ui-myprofile.caption",
+          "route" : "/settings/myprofile/changepw"
```

Comment:
Change password page have another url. So maybe should change it or create const with change password url and use it

Reply:
@OleksiiKuzminov, @VictorSoroka1 fixed